Code/Framework/AzCore/AzCore/Jobs/Internal/JobManagerWorkStealing.cpp

```diff
@@ -98,8 +98,7 @@ JobManagerWorkStealing::~JobManagerWorkStealing()
 
         for (ThreadInfo* thread : m_workerThreads)
         {
-            thread->m_waitEvent.release();
-            thread->m_thread.join();
```

Comment:
I think this join was there to finish executing those threads during shutdown.
Note the `m_quitRequested` being set here and the `if` on line 376
I think the original method is also unsafe... it should signal and wait for the thread to finish

Reply:
This was a typo, cleaning up the commit and everything now thanks